src/transport/CASESession.h

```diff
@@ -155,6 +221,14 @@ class DLL_EXPORT CASESession
 
     Protocols::SecureChannel::MsgType mNextExpectedMsg = Protocols::SecureChannel::MsgType::CASE_SigmaErr;
 
+    Hash_SHA256_stream mCommissioningHash;
```

Comment:
Are these used currently? I was wondering why the class needs an ongoing hash and extra data, but could not find usages.

Could we only add data members once used, so we can review usage and intent as well?

Reply:
The hash SHA256 stream is used to compute the salt for generating sigmaR2/3 and session Keys.
It is used to keep track of the previous Sigma messages that will be out of scope within the following Sigma message handling.
What you suggest is also an option, but will require us to store the sigma messages within the class's scope.